packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -339,271 +287,127 @@ export class AtomicSearchBox {
-      .flat();
-    const max =
-      this.numberOfQueries + elements.filter(elementHasNoQuery).length;
-
-    return elements.slice(0, max);
-  }
-
-  private focusValue(value: HTMLElement) {
-    this.updateActiveDescendant(value.id);
-    this.searchBoxCommon.scrollActiveDescendantIntoView();
-    this.updateQueryFromSuggestion();
-    this.updateAriaLiveActiveDescendant(value);
-  }
-
-  private focusPanel(panel: HTMLElement | undefined) {
-    if (this.panelInFocus === panel) {
-      return;
-    }
-    if (panel && panel.firstElementChild) {
-      const panelHasActiveDescendant =
-        this.previousActiveDescendantElement &&
-        panel.contains(this.previousActiveDescendantElement);
-      const newValue = panelHasActiveDescendant
-        ? this.previousActiveDescendantElement!
-        : (panel.firstElementChild as HTMLElement);
-      this.updateDescendants(newValue.id);
-      this.updateAriaLiveActiveDescendant(newValue);
-    }
-  }
-
-  private updateAriaMessage() {
-    const elsLength = this.allSuggestionElements.filter(elementHasQuery).length;
-    this.searchBoxAriaMessage = elsLength
-      ? this.bindings.i18n.t('query-suggestions-available', {
-          count: elsLength,
-        })
-      : this.bindings.i18n.t('query-suggestions-unavailable');
-  }
-
-  private async triggerSuggestions() {
-    if (this.isSearchDisabledForEndUser(this.searchBoxState.value)) {
-      return;
-    }
-    const settled = await Promise.allSettled(
-      this.suggestions.map((suggestion) =>
-        promiseTimeout(
-          suggestion.onInput ? suggestion.onInput() : Promise.resolve(),
-          this.suggestionTimeout
-        )
-      )
-    );
-
-    const fulfilledSuggestions: SearchBoxSuggestions[] = [];
-
-    settled.forEach((prom, j) => {
-      if (prom.status === 'fulfilled') {
-        fulfilledSuggestions.push(this.suggestions[j]);
-      } else {
-        this.bindings.engine.logger.warn(
-          'Some query suggestions are not being shown because the promise timed out.'
-        );
-      }
-    });
-
-    const splitSuggestions = (side: 'left' | 'right', isDefault = false) =>
-      fulfilledSuggestions
-        .filter(
-          (suggestion) =>
-            suggestion.panel === side || (!suggestion.panel && isDefault)
-        )
-        .sort(this.sortSuggestions);
-
-    this.leftSuggestions = splitSuggestions('left', true);
-    this.leftSuggestionElements = this.getAndFilterLeftSuggestionElements();
-
-    this.rightSuggestions = splitSuggestions('right');
-    this.rightSuggestionElements = this.getSuggestionElements(
-      this.rightSuggestions
-    );
-
-    const defaultSuggestedQuery =
-      this.allSuggestionElements.find(elementHasQuery)?.query || '';
-
-    this.updateSuggestedQuery(defaultSuggestedQuery);
-    this.updateAriaMessage();
+  private get searchBoxOptions(): SearchBoxOptions {
+    return {
+      id: this.id,
+      numberOfSuggestions: 0,
```

Comment:
Same question about setting this to 0

Reply:
Same answer as for insight search box. Note that this is not new code for the search box, merely moving code around.